<launch>

    <arg name="world_file" default="$(find rvo_ros)/simulation/worlds/multi_agents11.world"/>
    <arg name="gui" default="true"/>

    <rosparam command="load" file = "$(find rvo_ros)/configure/config.yaml"/>
    
    <node name="rvo_node" pkg="rvo_ros" type="rvo_node">
        <remap from="/rvo/model_states" to="/gazebo/model_states"/>
        <remap from="/rvo_vel" to="/obstacles_velocity"/>
    </node>

    <include file="$(find gazebo_ros)/launch/empty_world.launch">
        <arg name="use_sim_time" value="true"/>
        <arg name="debug" value="false"/>
        <arg name="gui" value="$(arg gui)"/>
        <arg name="world_name" value="$(arg world_file)"/>
        <arg name="verbose" value="false"/>
    </include>
    
</launch>
